---
# Documentation: https://wowchemy.com/docs/managing-content/

title: 'Interchange of DNA-binding Modes in the Deformed and Ultrabithorax Homeodomains:
  A Structural Role for the N-terminal Arm'
subtitle: ''
summary: ''
authors:
- Richard W Frazee
- Jennifer A Taylor
- Thomas D Tullius
tags:
- '"missing nucleoside"'
- '"hydroxyl radical footprinting"'
- '"DNA-protein"'
categories: []
date: '2002-01-01'
lastmod: 2020-10-27T15:44:26-04:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2020-10-27T19:44:26.435155Z'
publication_types:
- '2'
abstract: The deformed (Dfd) and ultrabithorax (Ubx) homeoproteins regulate developmental
  gene expression in Drosophila melanogaster by binding to specific DNA sequences
  within its genome. DNA binding is largely accomplished via a highly conserved helix-turn-helix
  DNA-binding domain that is known as a homeodomain (HD). Despite nearly identical
  DNA recognition helices and similar target DNA sequence preferences, the in vivo
  functions of the two proteins are quite different. We have previously revealed differences
  between the two HDs in their interactions with DNA. In an effort to define the individual
  roles of the HD N-terminal arm and recognition helix in sequence-specific binding,
  we have characterized the structural details of two Dfd/Ubx chimeric HDs in complex
  with both the Dfd and Ubx-optimal-binding site sequences. We utilized hydroxyl radical
  cleavage of DNA to assess the positioning of the proteins on the binding sites.
  The effects of missing nucleosides and purine methylation on HD binding were also
  analyzed. Our results show that both the Dfd and Ubx HDs have similar DNA-binding
  modes when in complex with the Ubx-optimal site. There are subtle but reproducible
  differences in these modes that are completely interchanged when the Dfd N-terminal
  arm is replaced with the corresponding region of the Ubx HD. In contrast, we showed
  previously that the Dfd-optimal site sequence elicits a very different binding mode
  for the Ubx HD, while the Dfd HD maintains a mode similar to that elicited by the
  Ubx-optimal site. Our current methylation interference studies suggest that this
  alternate binding mode involves interaction of the Ubx N-terminal arm with the minor
  groove on the opposite face of DNA relative to the major groove that is occupied
  by the recognition helix. As judged by hydroxyl radical footprinting and the missing
  nucleoside experiment, it appears that interaction of the Ubx recognition helix
  with the DNA major groove is reduced. Replacing the Dfd N-terminal arm with that
  of Ubx does not elicit a complete interchange of the DNA-binding mode. Although
  the position of the chimera relative to DNA, as judged by hydroxyl radical footprinting,
  is similar to that of the Dfd HD, the missing nucleoside and methylation interference
  patterns resemble those of the Ubx HD. Repositioning of amino acid side-chains without
  wholesale structural alteration in the polypeptide appears to occur as a function
  of N-terminal arm identity and DNA-binding site sequence. Complete interchange of
  binding modes was achieved only by replacement of the Dfd N-terminal arm and the
  recognition helix plus 13 carboxyl-terminal residues with the corresponding residues
  of Ubx. The position of the N-terminal arm in the DNA minor groove appears to differ
  in a manner that depends on the two base-pair differences between the Dfd and Ubx-optimal-binding
  sites. Thus, N-terminal arm position dictates the binding mode and the interaction
  of the recognition helix with nucleosides in the major groove.
publication: '*Journal of Molecular Biology*'
doi: 10.1016/s0022-2836(02)00996-8
---
