---
date: "2020-10-27"
external_link: ""
image:
  caption:
  focal_point: Smart
links:
- icon: twitter
  icon_pack: fab
  name: Follow
  url: https://twitter.com/tomtullius
slides: 
summary:
tags:
- RNA
title: A chemical probe of RNA structure at single-atom resolution
url_code: ""
url_pdf: ""
url_slides: ""
url_video: ""
---
{{< figure src="srl.png" title="**The sarcin/ricin loop (SRL) RNA molecule.** Colored in yellow are the ribose H5’/H5” hydrogen atoms that our work showed are abstracted by the hydroxyl radical." >}}
While RNA participates in nearly every aspect of cellular function, structural knowledge for most RNAs ends at their nucleotide sequence. This presents a challenge, because we know that biological function depends on three-dimensional structure.

Chemical and enzymatic probe experiments have been widely used to provide information on RNA structure. RNA is ideal for such experiments because of the hierarchical formation of secondary and tertiary structure that produces the functional three-dimensional structure of the RNA. There are now many experimental methods available for mapping RNA secondary structure (regions of sequence that are base-paired versus present in unpaired loops). 

Few probe methods are available for mapping RNA tertiary structure, which results from long-range contacts between secondary structure elements that result in a folded RNA molecule. One widely-used probe of tertiary structure is the hydroxyl radical. The hydroxyl radical abstracts a hydrogen atom from a ribose in the RNA backbone, causing a strand break. Ribose hydrogen atoms on the surface of RNA are abstracted more often than ribose hydrogens in the interior of the molecule. A quantitative map of hydroxyl radical-induced cleavage thus is a representation of the surface of the RNA molecule, at single-nucleotide resolution.

We recently published a paper that provides new insights into the chemical mechanism of hydroxyl radical cleavage of RNA. We discovered that the most abundant cleavage product is an RNA strand with a 5'-aldehyde terminus. Our work thus increases the resolution of the hydroxyl radical cleavage experiment to a single atom.

Until recently, chemical and enzymatic probe experiments were used to study one RNA at a time, using gel or capillary electrophoresis to produce a nucleotide-resolution structural map. But the advent of high-throughput DNA sequencing technology has now made it possible for a single experiment to produce structural information for large collections of RNA molecules in parallel, up to an entire transcriptome. While methods for secondary structure mapping of whole transcriptomes are now well established, there is no report of a transcriptome-wide experimental method for mapping RNA tertiary structure in living cells. A new project in the Tullius lab focuses on developing a high-throughput version of the hydroxyl radical cleavage experiment, with the aim of mapping RNA tertiary structure transcriptome-wide. 

This project is supported by a grant from the National Science Foundation (MCB-1616388).
