---
# Documentation: https://wowchemy.com/docs/managing-content/

title: The DNA binding specificity of engrailed homeodomain
subtitle: ''
summary: ''
authors:
- A Draganescu
- Thomas D Tullius
tags:
- '"DNA-protein"'
- '"hydroxyl radical footprinting"'
- '"missing nucleoside"'
categories: []
date: '1998-01-01'
lastmod: 2020-10-27T15:44:27-04:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2020-10-27T19:44:27.784571Z'
publication_types:
- '2'
abstract: The engrailed gene of Drosophila melanogaster is an integral member of the
  highly complex cascade which results in a fully developed fruitfly. The gene product
  of engrailed contains a homeodomain which is responsible for DNA binding via a helix-turn-helix
  motif. The crystal structure of this 60 amino acid residue domain complexed to DNA
  is analogous to structures of other homeodomain-DNA complexes, consistent with the
  high degree of sequence conservation within both protein and DNA. Despite the high
  degree of homology, homeodomains do exhibit distinct preferences for certain DNA
  sequences. Such specificity may be at least partly responsible for the interactions
  necessary for normal development. Using the hydroxyl radical as a chemical probe,
  we have examined complexes of Engrailed homeodomain with several DNA sequences to
  determine the protein's binding specificity in solution. We find that Engrailed
  forms a single, specific complex with a unique DNA binding site which is analogous
  to the complex seen in the co-crystal structure. In contrast, our chemical probe
  experiments show that the binding site of Engrailed that was determined by in vitro
  selection and that also was present in the co-crystal structure contains two possible
  binding sites. Modification of the sequence of this site to yield single binding
  sites removes the ambiguity, and results in two different, well-behaved Engrailed-DNA
  complexes. Our results underscore the utility of chemical probe experiments for
  defining the variety of modes of interaction of proteins with DNA that can occur
  in solution, but that might not be apparent in a crystal structure.
publication: '*Journal of Molecular Biology*'
doi: 10.1006/jmbi.1997.1567
---
